recipes/litespi/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "litespi" %}
+{% set version = "2023.12" %}
+{% set sha256 = "264666700b6a32e7b7cc9fac31c3f22ede47203267788b19ccae5f4e488153ce" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/litex-hub/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-build-isolation --no-deps
+  entry_points:
+    - litespi_gen = litespi.gen:main
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - litex >=2023.08
+    - liteiclink >=2023.08
+    - migen
+    - packaging
+    - pyyaml
+
+test:
+  imports:
```

Comment:
What about a pip check here?

Reply:
It doesn't specify its dependencies unfortunately https://github.com/litex-hub/litespi/blob/master/setup.py